### PR TITLE
ddl: Retry when session is done

### PR DESCRIPTION
**Pull request by zimulala**
Remove the `TODO`.

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @shenli @tiancaiamao @hanfei1991 @coocood

**Comment by coocood** [comment]
LGTM

File: ddl/owner_manager.go [hunk 1]
```diff
@@ -146,33 +151,42 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, key string, wg *sync.Wait
 	for {
 		select {
 		case <-m.etcdSession.Done():
-			// TODO: Create session again?
-			log.Warnf("[ddl] etcd session is done.")
+			log.Info("[ddl] etcd session is done, creates a new one")
+			err := m.newSession(ctx, newSessionRetryUnlimited)
+			if err != nil {
+				log.Infof("[ddl] break %s campaign loop, err %v", key, err)
```
Comment (shenli) [inline ddl/owner_manager.go]: Should we use warn here? And should we backoff when create session failed.

Comment (zimulala) [inline ddl/owner_manager.go]: I will change it to warn level.
I think backoff isn't necessary because we will retry util `context` is canceled.

**Comment by shenli** [comment]
@zimulala Please resolve the conflicts.

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**